operator/config/crd/bases/core.cnrm.cloud.google.com_configconnectorcontexts.yaml

```diff
@@ -97,6 +97,12 @@ spec:
                 - Absent
                 - Merge
                 type: string
+              version:
+                description: |-
+                  Version specifies the exact addon version to be deployed, eg 1.2.3
```

Comment:
Today, if they get it wrong it's an error (it will fail to load the manifest)

Reply:
We can add a kubebuilder enum (though this may increase the chance that we take the blame if CC release messes it up), and also include the available CC version in the release note.

Reply:
I like the idea of including valid values in the release note.

I don't think we should use an enum, as long as the error is reasonable.  I'm also willing to tolerate improving the error over time.

Reply:
I think as long as the .status.error relfects a problem without being too unpleasant to read, then I'm fine with it. Please lets run that test.